modules/ROOT/pages/microsoft-azure-sb-connector.adoc

```diff
@@ -0,0 +1,334 @@
+
+For more https://docs.mulesoft.com/mule-runtime/3.7/flow-processing-strategies#fine-tuning-a-queued-asynchronous-processing-strategy.
+
+=== Random Errors when Using a Manual ACK
+
+You might receive errors like this one when working with a high number of messages while a using manual ACK:
+
+```
+[com.mulesoft.connectors.microsoft.azure.servicebus.internal.error.exception.ConsumerException: Message with lockToken 2dc1312f-b263-4282-bbb0-566998eff6e6 could not be ACK at com.mulesoft.connectors.microsoft.azure.servicebus.internal.connection.Connection.ack(Connection.java:192)][ERROR 2019-05-22 00:17:30,822 [ReactorThread6f355ff5-5a36-487b-bb70-1a995a6ddf74] com.microsoft.azure.servicebus.primitives.CoreMessageReceiver: Completing pending updateState operation for delivery '?     ? &??I??=????' with exception com.microsoft.azure.servicebus.primitives.MessageLockLostException: The lock supplied is invalid. Either the lock expired, or the message has already been removed from the queue.]
+```
+When the connector performs a 'prefetch', the lockToken's validity time is fixed in relation to that moment. The problem arises when the lockToken's validity time is not long enough to process the entire batch of records. In these cases, Mule might throw an error because the lockToken expired before you do an ACK.
+
+*Reviewers, where do you do this?*
+
+To prevent this issue, reduce the size of the prefetch (default to 100), and increase the validity time of the lockToken, or both. You can do this when creating the queue/subscription from the connector, in the lock duration property or from Azure portal for existing queues). The maximum duration value for the lock token in Azure is 5 minutes (300 seconds).
+
+=== Reduce Noise in Mule Apps Logs
+
+In some circumstances, the underlying library used by the connector might regularly log complete stacktraces with level WARN. These messages do not represent an issue, but they can clutter the logs. To reduce the noise in the logs, make either of the following changes to the `src/main/resources/log4j2.xml` file:
+
+* Specify an *AsyncLogger* for the library's package, raising the level of the log:
+
+```
+<!-- Recommended for packages not belonging to Mule -->
+<AsyncLogger name="com.microsoft.azure.servicebus" level="ERROR"/>
+```
+
+* Add a *RegexFilter* to the existing appender (E.g. RollingFile):
+
+```
+<!-- Avoids the log of messages that contain the specified regexp -->
+<RegexFilter regex = ".message to filter.*" onMatch="DENY" onMismatch="ACCEPT"/>
+```
+IMPORTANT: If the message is multiline, RegexFilter might not work correctly.
+
+=== Listeners (Sources) in Network Disconnection and Reconnection Scenarios
+
+In situations were there are network connection problems, you can direct the source components for the Azure connector to automatically execute reconnection attempts by specifying a reconnection strategy. You can specify a reconnection strategy when you create or edit the connector configuration:
```

Comment:
In situations were there are > When there are,

Is it okay to call this only "Azure connector"?